README.md

```diff
@@ -1,7 +1,7 @@
 # Drone plugin for Helm 3
 
 [![Build Status](https://cloud.drone.io/api/badges/pelotech/drone-helm3/status.svg)](https://cloud.drone.io/pelotech/drone-helm3)
```

Comment:
Build status should be removed, or updated to our drone.  Not sure what the status of allowing build badges is at.

Reply:
will do, thx. I think badges are allowed now, as far as I remember

Reply:
updated the batch and adjusted other parts of Readme